admin-dev/themes/new-theme/js/pages/product/components/generator/AttributesSelector.vue

```diff
@@ -120,31 +120,21 @@
 </template>
 
 <script lang="ts">
-  import Vue, {PropType, VueConstructor} from 'vue';
+  import {defineComponent, PropType} from 'vue';
   import isSelected from '@pages/product/mixins/is-attribute-selected';
   import ProductMap from '@pages/product/product-map';
   import PerfectScrollbar from 'perfect-scrollbar';
   // @ts-ignore
   import Bloodhound from 'typeahead.js';
   import AutoCompleteSearch, {AutoCompleteSearchConfig} from '@components/auto-complete-search';
   import Tokenizers from '@components/bloodhound/tokenizers';
-  import {Attribute, AttributeGroup} from '@pages/product/types';
+  import {Attribute, AttributeGroup, AttributesSelectorStates} from '.';
```

Comment:
Why remove the `types` file? It seemed to be more convenient to import the types into any files for the App. Now it's bound in the index files so you can't easily import the types independently from the App itself.